master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -151,107 +111,148 @@ spec:
+    matchLabels: {}
+  policyTypes:
+  - Egress
+EOF
 ```
 
-### Prevent outgoing connections from pods
+#### Verify Access - Denied All Egress
 
-Kubernetes NetworkPolicy does not provide a way to prevent outgoing connections from pods.  However,
-Calico does. In this section we'll create egress policies using `calicoctl`
-that allow the outgoing connections in the `advanced-policy-demo` namespace we
-want; all other egress traffic will be denied.
+Now any ingress or egress traffic which is not explicitly allowed by a policy will be denied.
 
-To do this, we'll need to create a policy which selects all pods in the namespace, and allows
-egress traffic to other pods in the namespace.
+We can see that this is the case by switching over to our "access" pod in the
+namespace and attempting to lookup nginx or access google.com.
 
+```shell
+/ # nslookup nginx
+Server:    10.96.0.10
+Address 1: 10.96.0.10
+
+
+nslookup: can't resolve 'nginx'
+/ # wget -q --timeout=5 google.com -O -
+wget: bad address 'google.com'
 ```
-calicoctl apply -f - <<EOF
-apiVersion: v1
-kind: policy
+
+> **Note**: The `nslookup` command can take a minute or more to timeout.
+{: .alert .alert-info}
+
+### 5. Allow DNS egress traffic
+
+Run the following to create a `NetworkPolicy` which allows DNS egress traffic
+from any pods in the `advanced-policy-demo` namespace.
+
+```shell
+kubectl create -f - <<EOF
+apiVersion: networking.Kubernetes.io/v1
+kind: NetworkPolicy
 metadata:
-  name: advanced-policy-demo.allow-egress
+  name: allow-dns-access
+  namespace: advanced-policy-demo
 spec:
-  selector: calico/k8s_ns == 'advanced-policy-demo'
-  order: 500
+  podSelector:
+    matchLabels: {}
+  policyTypes:
+  - Egress
   egress:
-  - action: allow
-    destination:
-      selector: calico/k8s_ns == 'advanced-policy-demo'
+  - to:
+    ports:
+    - protocol: UDP
+      port: 53
 EOF
 ```
 
-Notice that we've specified an order of 500.  This means that this policy will be applied before any
-of the Kubernetes policies.
+#### Verify Access - Allowed DNS access
+
+Now egress traffic to DNS will be allowed.
 
-We also need to create a policy which allows traffic to access kube-dns. Let's create one now in the kube-system namespace.
-We'll specify an order of 500 so that it also takes precedence over the Kubernetes policies.
+We can see that this is the case by switching over to our "access" pod in the namespace and attempting to lookup nginx and google.com.
 
+```shell
+/ # nslookup nginx
+Server:    10.0.0.10
+Address 1: 10.0.0.10 kube-dns.kube-system.svc.cluster.local
+/ # nslookup google.com
+Name:      google.com
+Address 1: 2607:f8b0:4005:807::200e sfo07s16-in-x0e.1e100.net
+Address 2: 216.58.195.78 sfo07s16-in-f14.1e100.net
 ```
-calicoctl apply -f - <<EOF
-apiVersion: v1
-kind: policy
+
+Even though DNS is now working, all egress traffic from all pods in the advanced-policy-demo namespace is still blocked other than DNS.  Therefore the `wget` calls will still fail.
```

Comment:
> Therefore the `wget` calls will still fail.

Could you add some more clarification to this statement - e.g. "Therefore the `wget` calls from the access pod to both google.com and the nginx service will still fail"